WcaOnRails/app/models/round.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+class Round < ApplicationRecord
+  belongs_to :competition_event
+  has_one :event, through: :competition_event
+  belongs_to :format
+
+  MAX_NUMBER_OF_ROUNDS_PER_EVENT = 4
```

Comment:
Ok, we're going with `MAX_NUMBER`! This is only used in 1 place right now, but I expect it will be used in our UI code later on. Tempted to stick with the variable as it is right now. (Feel free to YAGNI this argument.)